crates/sui-indexer-alt-graphql/src/api/types/transaction_kind/programmable/commands/transaction_argument.rs

```diff
@@ -52,3 +61,23 @@ impl From<NativeArgument> for TransactionArgument {
         }
     }
 }
+
+impl From<proto::Argument> for TransactionArgument {
```

Comment:
To go with my suggestion above, you can implement this as `TryFrom<proto::Argument>` where `Error = ()`.